website/src/docs/reference/config.md

```diff
@@ -68,3 +68,132 @@ option_3: foo # Taken from $XDG_CONFIG_HOME/task/.taskrc.yml
 The experiments section allows you to enable Task's experimental features. These
 options are not enumerated here. Instead, please refer to our
 [experiments documentation](../experiments/index.md) for more information.
+
+```yaml
+experiments:
+  feature_name: 1
+  another_feature: 2
+```
+
+### `remote` <Badge type="warning" text="Remote experimentation" />
+
+- **Type**: `object`
+- **Description**: Remote configuration settings for handling remote Taskfiles
+- **Requires**: [Remote Taskfiles experiment](../experiments/remote-taskfiles.md) to be enabled
+
+```yaml
+remote:
+  insecure: false
+  offline: false
+  timeout: "30s"
+  cache-expiry: "24h"
+```
+
+#### `insecure`
+
+- **Type**: `boolean`
+- **Default**: `false`
+- **Description**: Allow insecure connections when fetching remote Taskfiles
+- **CLI equivalent**: [`--insecure`](./cli.md#--insecure)
+
+```yaml
+remote:
+  insecure: true
+```
+
+#### `offline`
+
+- **Type**: `boolean`
+- **Default**: `false`
+- **Description**: Work in offline mode, preventing remote Taskfile fetching
+- **CLI equivalent**: [`--offline`](./cli.md#--offline)
+
+```yaml
+remote:
+  offline: true
+```
+
+#### `timeout`
+
+- **Type**: `string`
+- **Default**: Not specified
+- **Pattern**: `^[0-9]+(ns|us|µs|ms|s|m|h)$`
+- **Description**: Timeout duration for remote operations (e.g., '30s', '5m')
+- **CLI equivalent**: [`--timeout`](./cli.md#--timeout-duration)
+
+```yaml
+remote:
+  timeout: "1m"
+```
+
+#### `cache-expiry`
+
+- **Type**: `string`
+- **Default**: Not specified
+- **Pattern**: `^[0-9]+(ns|us|µs|ms|s|m|h)$`
+- **Description**: Cache expiry duration for remote Taskfiles (e.g., '1h', '24h')
+- **CLI equivalent**: [`--expiry`](./cli.md#--expiry-duration)
+
+```yaml
+remote:
+  cache-expiry: "6h"
+```
+
+----
+
+### `verbose`
+
+- **Type**: `boolean`
+- **Default**: `false`
+- **Description**: Enable verbose output for all tasks
+- **CLI equivalent**: [`-v, --verbose`](./cli.md#-v---verbose)
+
+```yaml
+verbose: true
+```
+
+### `concurrency`
+
+- **Type**: `integer`
+- **Minimum**: `1`
+- **Description**: Number of concurrent tasks to run
+- **CLI equivalent**: [`-C, --concurrency`](./cli.md#-c---concurrency-number)
+
+```yaml
+concurrency: 4
+```
+
+## Example Configuration
+
+Here's a complete example of a `.taskrc.yml` file with all available options:
+
+```yaml
+# Enable experimental features
+experiments:
+  REMOTE_TASKFILES: 1
+
+# Remote configuration
+remote:
+  insecure: false
+  offline: false
+  timeout: "30s"
+  cache-expiry: "1h"
+
+# Global settings
+verbose: true
+concurrency: 2
+```yaml
```

Comment:
Think this got accidentally doubled up. Suggested changes are annoying sometimes 😆